tests/functional/models/test_slate_lineup_model.py

```diff
@@ -69,36 +107,8 @@ async def test_get_slate_lineup(self, slate_config, slate_lineup_config):
     @patch.object(SlateConfigModel, 'SLATE_CONFIGS_BY_ID', slate_configs_by_id)
     @patch.object(app.config, 'qa_slate_map', {slate_config_id: slate_config_id_2})
     async def test_get_slate_lineup_for_qa_user(self, slate_lineup_config):
-        self.candidateSetTable.put_item(Item={
-            "id": "test-candidate-id",
-            "version": 1,
-            "created_at": 1612907252,
-            "candidates": [
-                {
-                    "feed_id": 1,
-                    "item_id": 3208490410,
-                    "publisher": "hbr.org"
-                }
-            ]
-        })
-
-        self.candidateSetTable.put_item(Item={
-            "id": "test-candidate-id-2",
-            "version": 1,
-            "created_at": 1612907252,
-            "candidates": [
-                {
-                    "feed_id": 1,
-                    "item_id": 11,
-                    "publisher": "getpocket.com"
-                },
-                {
-                    "feed_id": 1,
-                    "item_id": 12,
-                    "publisher": "getpocket.com"
-                }
-            ]
-        })
+        self.candidateSetTable.put_item(Item=test_candidate)
+        self.candidateSetTable.put_item(Item=test_candidate_2)
```

Comment:
Minor, but in Python we name variables `like_this` rather than `likeThis`, and right now we have both of these naming styles going on in different parts of this file